src/olympia/api/urls.py

```diff
@@ -94,6 +102,13 @@ def get_versioned_api_routes(version, url_patterns):
     re_path(r'^shelves/', include('olympia.shelves.urls')),
 ]
 
+url_name = 'schema'
+
+try:
+    static_swagger_ui_js = staticfiles_storage.url('js/swagger/swagger_ui.js')
+except ValueError:
+    static_swagger_ui_js = None
+
```

Comment:
Should this be there? This is duplicated from what's in `get_versioned_api_routes()` except without the versioning